bot/client.js

```diff
@@ -10,13 +11,40 @@ const client = new Discord.Client();
 // Pre-load controllers
 const controllers = require('./controllers')();
 
+// Award bonus points
+const awardBonusPoints = async (user) => {
+  const numberOfMessagesForBonus = 1000;
+  const amountOfBonusPoints = 100;
+  // Get User Message Count
+  const memberData = await models.Member.findAll(
+    {
+      attributes: ['messagesCount', 'points'],
+      where: { discordUser: user.author.id },
+    },
+  );
+  let { messagesCount } = memberData[0].dataValues;
+  let { points } = memberData[0].dataValues;
```

Comment:
Hey! If you wanted to you could condense these two lines into one. When deconstructing a variable, you can add multiple items... For example here you could say `let { messagesCount, points } = memberData[0].dataValues;` and it would say the same thing with one less line of code. They could also be made a const too since they don't change.